dog.js

```diff
@@ -0,0 +1,57 @@
+
+function murphy() {
+    alert('Murphy, Boldest of Boys, $123.45');
+  }
+
+function poppy() {
+    alert('Poppy, Name Taker, $123.45');
+  }
+
+function jack() {
+    alert('Jack, Irredeemable Poet, $123.45')
+}
+
+function duffy() {
+    alert('Duffy, Criminal Lord of All, $123.45')
+}
+
+function lucas() {
+    alert('Lucas, Soulful Stare, $123.45')
+}
+
+function jake() {
+    alert('Jake, Regal like a theater, $123.45')
+}
+
+function angus() {
+    alert('Angus, Wreck you up, $123.45')
+}
+
+function violet() {
+    alert('Violet, Smaller Doggo, $123.45')
+}
+
+function piper() {
+    alert('Piper, Absolute Lapper, $123.45')
+}
+
+function maximus() {
+    alert('Maximus, Take you to court, $123.45')
+}
+
+function luna() {
+    alert('Luna, So Smiley, $123.45')
+}
+
+function stella() {
+    alert('Stella, Smallo Doggo, $123.45')
+}
+
+let runningTotal = 0;
+
+function cost(price) {
+    let total = 0;
+    total += price;
+    runningTotal += price;
```

Comment:
Lines 53 & 54 are redundant since you are ultimately adding it to the variable runningTotal and the variable total is not being used.